addons/common/fnc_isTurnedOut.sqf

```diff
@@ -6,74 +6,78 @@ Description:
     Checks whether a unit is turned out in a vehicle or not.
 
 Parameters:
-    _unit - Unit to check [Object]
+    _unit - Unit to check <OBJECT>
 
 Returns:
-    "true" for turned out or "false" for not turned out [Boolean]
+    "true" for turned out or "false" for not turned out <BOOL>
     Cargo in exposed vehicles are "turned out" if they lack a
     cargoSoundAttenuation matching index. Do NOT assume that because
     this function returns false that the unit is generally exposed.
 
-    For example, ATVs will return false, even though they are "turned
-    out".
-
 Examples:
     (begin example)
-    if ( [player] call CBA_fnc_isTurnedOut ) then
-    {
-        player sideChat "I am turned out!";
-    };
+        if (player call CBA_fnc_isTurnedOut) then {
+            player sideChat "I am turned out!";
+        };
     (end)
 
 Author:
-    Nou, courtesy of ACRE project.
     shukari
 ---------------------------------------------------------------------------- */
-SCRIPT(isTurnedOut);
 
 params ["_unit"];
 
 if (isNull objectParent _unit) exitWith {true};
 if (isTurnedOut _unit) exitWith {true};
 
-private _return = false;
 private _vehicle = vehicle _unit;
-private _role = _unit call CBA_fnc_vehicleRole;
-
-// FFVs are allways outside
-if ([_unit] call ace_common_fnc_getTurretIndex in ([_vehicle] call ace_common_fnc_getTurretsFFV)) then {_return = true};
-
-// Spezial condition for RHS car windows
-if (_vehicle isKindOf "rhsusf_m1025_w" && {_role in ["driver", "Turret"]}) then {
-    private _cargoIndex = (_vehicle getCargoIndex _unit) min 2;
-    private _window = ['ani_window_1', 'ani_window_2', 'ani_window_4', 'ani_window_3'] select (_cargoIndex + 1);
-    _return = (_vehicle doorPhase _window) > 0;
+private _role = toLower (_unit call CBA_fnc_vehicleRole);
+private _isFFV = fullCrew [_vehicle, "turret"] findIf {_x select 0 == _unit && {_x select 4}} != -1;
+
+// FFVs are always outside
+if (_isFFV) exitWith {true};
+
+// special condition for RHS car windows
+if (_vehicle isKindOf "rhsusf_m1025_w" && {_role in ["driver", "turret"]}) exitWith {
```

Comment:
if the return of this exitWith is false... i must run the driver and turret exitWiths